src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Traffic Manager Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Traffic Manager management functions for managing the Microsoft Azure Traffic Manager service.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.1.0.0")]
+[assembly: AssemblyVersion("2.2.0.0")]
```

Comment:
@vladca please keep `AssemblyVersion` at 2.0.0.

`AssemblyFileVersion` was the only version that should have changed, so it can remain at 2.2.0